src/main/resources/se/diabol/jenkins/pipeline/DeliveryPipelineView/configure-entries.jelly

```diff
@@ -27,6 +27,10 @@
             <f:select/>
         </f:entry>
 
+        <f:entry title="Max number of pipelines" field="maxNumberOfJobs" default="-1">
+            <f:textbox/>
+        </f:entry>
```

Comment:
@tommysdk Then we should think of refactor and rename the ComponentComparator class and make an option for it to limit the result.